File: Backend/FormationService/FormationService/Models/Formation.cs
namespace FormationService.Models
{
    public class Formation
    {
        public int FormationId { get; set; }
        public string FormationName { get; set; }
        public required string SchoolName { get; set; }

        public required string Description { get; set; }
        public required ICollection<ModuleFormation> ModuleFormations { get; set; }
    }
}

File: app/model/formation.ts

import {Module} from './module';

export class Formation {


  formationId: number = 0; // correspond à FormationId
  formationName: string = ""; // correspond à FormationName
  schoolName: string = ""; // correspond à SchoolName
  description: string = ""; // correspond à Description
  modules: Array<Module> = []; // correspond à ModuleFormations



}


export interface PageFormation {

  formations : Formation[];
  page : number ;
  size : number ;
  totalpages : number ;
}
